### PR TITLE
Add export to UG

### PR DESCRIPTION
**Pull request by yanchenglee98**
Add export function details to UG

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Lee Yan Cheng
 
-<img src="images/leeyancheng.png" width="200px">
```
Comment (yanchenglee98) [inline docs/AboutUs.md:34]: tp progress script requires filename to be github username

File: docs/UserGuide.md [hunk 2]
```diff
@@ -150,7 +150,13 @@ Format: `exit`
...
 ### Saving the data
 
-Hall-y's data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+
+### Exporting of email : `export`
+
+Exports the email address of all entries in Hall-y as a .txt file.
+
+Format: `export`
```
Comment (jieffyy) [inline docs/UserGuide.md]: Hmm seems like there was an issue with Git. Could you keep the original line under 'Saving the Data'?

Also, could you update the command summary at the bottom of the UG as well?

Comment (yanchenglee98) [reply docs/UserGuide.md]: Yep will do thanks

**Review by jieffyy — APPROVED** [review]
Okay, LGTM 👍 . Merging!

**Pull request merged**